### PR TITLE
HC-950 How to do a loopback test

### PR DESCRIPTION
**Pull request by seaxwi**
**Changes proposed in this pull request**
Complete list of compatible boards, pin diagrams, and post-test instructions.

### CODE REVIEW

**Review by Renat0Ribeir0 — APPROVED** [review]
LGTM!

File: content/Hardware/Generic/How-to-do-a-loopback-test.md [hunk 1]
```diff
@@ -24,6 +22,12 @@ A failed loopback test indicates a problem with one or more of the following:
 
 4. Connect a jumper cable from **Digital 0 (RX)** to **Digital 1 (TX)**.
 
+   ![](img/loopback-circuit-uno.png)
+   _Pin connections (orange) for Arduino UNO._
```
Comment (jojo1212) [inline content/Hardware/Generic/How-to-do-a-loopback-test.md:26]: The pin connection looks confusing as it looks like all 4 pins (reset, 3v3, 5v, and gnd) have to be shorted together.

File: content/Hardware/Generic/How-to-do-a-loopback-test.md [hunk 2]
```diff
@@ -2,19 +2,17 @@
 title: "How to do a loopback test"
 ---
 
+A **loopback test** is a troubleshooting procedure to test the serial communication between the PC and Arduino board with a dedicated USB to Serial converter chip. It can be performed with the following boards:
 
-For Arduino boards with a dedicated USB to serial adapter chip (Uno, Mega, or Nano), a *loopback test* can be performed to test the serial communication between the board and your PC.
```
Comment (jojo1212) [inline content/Hardware/Generic/How-to-do-a-loopback-test.md:6]: We will have to test and confirm on the different versions of the board just to be sure it works.

**Review by jojo1212 — COMMENTED** [review]
We will have to test the process on different boards and address the comments attached to [HC-950](https://arduino.atlassian.net/browse/HC-950?atlOrigin=eyJpIjoiZDU4MGZlMmQ3MzFlNDYzMWI1ZGExNGFkMzY0ZGZlNTkiLCJwIjoiaiJ9).

File: content/Hardware/Generic/How-to-do-a-loopback-test.md (hunk 2 above)
Comment (seaxwi) [reply content/Hardware/Generic/How-to-do-a-loopback-test.md:6]: @jojo1212 I've tested the Rev3 SMD. I can look in the lab for the Mega variants.

File: content/Hardware/Generic/How-to-do-a-loopback-test.md (hunk 1 above)
Comment (seaxwi) [reply content/Hardware/Generic/How-to-do-a-loopback-test.md]: Fixed

File: content/Hardware/Generic/How-to-do-a-loopback-test.md [hunk 3]
```diff
@@ -34,8 +38,30 @@ A failed loopback test indicates a problem with one or more of the following:
...
    ![Performing a loopback test in the Arduino IDE serial monitor.](img/serial_monitor_loopback_test.gif "Performing a loopback test in the Arduino IDE serial monitor")
 
-  *Performing a loopback test in the Arduino IDE serial monitor.*
+   *Performing a loopback test in the Arduino IDE serial monitor.*
+
+## If the loopback test failed
+
+A failed loopback test failed indicates that the USB to TTL serial adapter is damaged and that the board should be replaced. [Contact us](https://www.arduino.cc/en/contact-us/) for questions about warranty, or visit [the Arduino Store](https://store.arduino.cc/).
+
+> **Note:** This test will always fail for the CH340 USB to serial chip used on some derivate boards.
```
Comment (jojo1212) [inline content/Hardware/Generic/How-to-do-a-loopback-test.md:51]: Should we add the link to How to spot counterfeit board article here?

Comment (seaxwi) [reply content/Hardware/Generic/How-to-do-a-loopback-test.md:51]: I would say no, as the relevant difference is using a different chip, regardless of whether the derivate would be considered a counterfeit.

File: content/Hardware/Generic/How-to-do-a-loopback-test.md (hunk 2 above)
Comment (seaxwi) [reply content/Hardware/Generic/How-to-do-a-loopback-test.md:6]: I couldn't find a Arduino Mega ADK Rev3 but otherwise I think we're covered.

Comment (jojo1212) [reply content/Hardware/Generic/How-to-do-a-loopback-test.md:6]: We will keep a note of this and whenever we will be able to get our hands on this board we will try it. I will check with David to see if he has any that can be borrowed.

**Review by jojo1212 — APPROVED** [review]
Everything looks good then.

**Pull request merged**